tests/slab.rs

```diff
@@ -659,3 +659,14 @@ fn drain_rev() {
     let vals: Vec<u64> = slab.drain().rev().collect();
     assert_eq!(vals, (0..9).rev().collect::<Vec<u64>>());
 }
+
+#[test]
+fn try_remove() {
+    let mut slab = Slab::new();
+
+    let key = slab.insert(1);
+
+    assert_eq!(slab.try_remove(key), Some(1));
+    assert_eq!(slab.try_remove(key), None);
+    assert!(slab.get(key).is_none());
```

Comment:
Is there any narrative intent behind using `Eq` in one assert, and `.is_none()` in the other, vs using either one consistently throughout?

Reply:
Thank you for pointing this out. The PR is updated